src/lib/modernExtend.ts

```diff
@@ -751,3 +778,74 @@ export function pressure(args?: Partial<NumericArgs>): ModernExtend {
         ...args,
     });
 }
+
+export function illuminance(args?: Partial<NumericArgs>): ModernExtend {
```

Comment:
Please also add to the definition generator: https://github.com/Koenkk/zigbee-herdsman-converters/blob/72c45d9ce26fafd4f6df5ca99e8c6fac17df097d/src/lib/generateDefinition.ts#L166

Reply:
Done